package.json

```diff
@@ -28,7 +28,8 @@
     "debounce": "^1.2.1",
     "jquery": "^3.7.0",
     "normalize.css": "^8.0.1",
-    "stimulus-autocomplete": "^3.1.0"
+    "stimulus-autocomplete": "^3.1.0",
+    "tippy.js": "^6.3.7"
```

Comment:
We get `popper` (an older version) already with admintlte/bootstrap 4, and need to activate it before using.

https://adminlte.io/docs/3.2/dependencies.html
https://getbootstrap.com/docs/4.1/components/tooltips/
https://popper.js.org/docs/v1/

Did you try that before using `tippy`? I don't think we have an existing example to point at, but wanted to call it out before we add another dependency

Reply:
neat, a little less ergonomic but its there. 95efc0dbdd9f167abcce85a7e00e644368d4d46a